modules/special-rules/pages/dauntless.adoc

```diff
@@ -0,0 +1,9 @@
+= Dauntless
+
+Some troops are so disciplined that they never flinch when facing the enemy -- such as the Dark Angels Deathwing, or Tyranid Swarms with Synapse Creatures.
+
+---
+
+Apply these rules to detachments with the {dauntless} ability:
+
+* The detachment automatically passes all Leadership tests.
```

Comment:
To follow through on the separation I try to keep between in-universe stuff and player agency, and to work in active case/second person: 
```suggestion
Where your detachment has the {dauntless} ability:

* You automatically pass any Leadership test that you are required to make for this detachment.
```